xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -893,6 +893,13 @@ void CVideoPlayer::OpenDefaultStreams(bool reset)
 
   bool valid;
 
+  if (m_pDemuxer && m_SelectionStreams.Count(STREAM_VIDEO) > 0)
```

Comment:
i would drop the 2nd condition. demuxer may have video at a later time and the call does not harm if there are no video streams.